### DISCUSSION TITLE
how to pbypass SSL errors in connecting

### DISCUSSION THREAD

**Discussion by fursaanva** [drygdryg/netbox-device-autodiscovery#3] [Q&A]
hi,
how to bypass the SSL check for connection to netbox

I get the following error.

requests.exceptions.SSLError: HTTPSConnectionPool(host='localhost', port=443): Max retries exceeded with url: /api/dcim/device-roles/?name=PC (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1131)')))

**Comment by drygdryg** [comment]
Try disabling SSL certificate verification: https://stackoverflow.com/questions/62773263/does-pynetbox-api-have-an-option-to-verify-ca-cert-for-self-signed-cert
Alternatively, you can try to install your self-signed certificate on the machine where you run the script: https://wiki.archlinux.org/title/User:Grawity/Adding_a_trusted_CA_certificate
